### ISSUE TITLE
Tailwind opacity removed in production since 1.2.0

### ISSUE THREAD

**Issue by stephanvierkant** [symfony/webpack-encore#982]
I've upgraded to Webpack Encore `1.2.0` and in production mode the opacity in Tailwindcss doesn't work. With Encore `1.0.3` is works fine.

```
.foo {
  @apply bg-black bg-opacity-70;
}
```
becomes:

```
.foo {
    --tw-bg-opacity: 0.7;
    background-color: #000;
}
```

I can see some dependencies have been updated in [package.json](https://github.com/symfony/webpack-encore/compare/v1.1.2...v1.2.0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519), but I haven't figured out which dependency causes this issue. 

Anyone with a similar experience?

**Comment by stof** [comment]
Well, css-minimizer-webpack-plugin has been updated to use cssnano 5 instead of cssnano 4. Maybe the new version has a bug.

**Comment by stephanvierkant** [comment]
I haven't found the root cause, but the issue has been fixed by updating to Encore 1.4.0.
